src/databases/Mili/avtMiliFileFormat.C

```diff
@@ -3352,7 +3352,12 @@ avtMiliFileFormat::ExtractJsonClasses(rapidjson::Document &jDoc,
     int cCount = 0;
     if (jClasses.HasMember("count"))
     {
-        cCount = jClasses["count"].GetInt();
+        //
+        // Counts may be in scientific notation. We need to retrieve 
+        // as a double and cast to int. 
+        //
+        double dbl = jClasses["count"].GetDouble();
+        cCount = (int) dbl;
```

Comment:
Quick question, since this has been brought up in other scenario's in VisIt. Are you sure an 'int' is large enough to hold the value? Would it be better to use an int64 type?

Reply:
I was actually thinking about this too... I'm pretty sure all of these should be in range, but I do know that the number of time steps can get fairly large with mili. I'm going to make a note to talk with Kevin next week to get an idea of what this range is. Thanks for bringing this up!

Reply:
I spoke with Kevin about this, and it looks like int should be large enough for these values.